bridge/slack/helpers_test.go

```diff
@@ -0,0 +1,26 @@
+package bslack
+
+import (
+	"testing"
+)
+
+func TestExtractTopicOrPurpose(t *testing.T) {
+	tables := []struct{
+		input string
+		changeType string
+		output string
+	}{
+		{"@someone set channel topic: one liner",    "topic",   "one liner"},
+		{"@someone set channel purpose: one liner",  "purpose", "one liner"},
+		{"@someone set channel topic: multi\nliner", "topic",   "multi\nliner"},
+		{"@someone cleared channel topic",           "topic",   ""},
+		{"some unmatched message",                   "unknown", ""},
+	}
+
+	for _, table := range tables {
+		changeType, output := extractTopicOrPurpose(table.input)
+		if changeType != table.changeType || output != table.output {
```

Comment:
You should be using the `assert.Equal` methods here from the `assert` package that is already vendored in. You can see an example in the test that I recently added in the `bridge/helper` package. :)